render.sh

```diff
@@ -16,7 +17,7 @@ if [[ -z $ORG && -z $REPO ]]; then
   REPO=${full#*/} # bash expansion: delete shortest match before '/'
   REPO=${REPO%/*} # bash expansion: delete shortest match after '/'
 fi
-if [[ -z "${FORECASTS}" && -e "predtimechart-config.yml" ]]; then
+if [[ -z "${FORECASTS}" && -e "/site/predtimechart-config.yml" ]]; then
```

Comment:
At some point, I'd benefit from talking through what's happening over here: https://github.com/hubverse-org/hub-dashboard-control-room/blob/9d364f673aa4b889c4ea88faace1b8495cc912af/.github/workflows/generate-site.yaml#L35. We're doing some checkout actions, etc. in that workflow and saying that whatever working directory github has should be mounted as `/site/` in the container?  Do those checkout steps not actually happen inside of the container?

Reply:
This is related to #9 


> We're doing some checkout actions, etc. in that workflow and saying that whatever working directory github has should be mounted as /site/ in the container? 

Correct. 

> Do those checkout steps not actually happen inside of the container?

The checkout steps do happen inside of the container. That's what the `container:` key is for in the job. I get a feeling that behind the scenes, GitHub is using docker-compose to combine our container with their run environment.

Reply:
Which is to say, you have a good point. After this, I'm going to clean things up and fix #9. When that happens, the scripts will live in `/bin` and the working directory will default to `/scripts`, so I will be able to remove the string.

Reply:
Thanks, that's helpful